posthog/tasks/scheduled.py

```diff
@@ -105,6 +106,12 @@ def setup_periodic_tasks(sender: Celery, **kwargs: Any) -> None:
         name="warm team access caches",
     )
 
+    sender.add_periodic_task(
```

Comment:
there's a scheduled task with expiry helper we could use here
if celery does it's usual "not running jobs" malarkey we don't care about running the missed ones, right?

so we could set an expiry of 50 minutes 
almost never helpful but might help avoid a thundering herd of queries after an outage